sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/message.py

```diff
@@ -91,7 +92,9 @@
         'group_id',
         'group_sequence',
         'reply_to_group_id'
-    ])
+    ],
+    defaults=(None,) * 13 # type: ignore
+    )
 Properties._code = 0x00000073 # type: ignore # pylint:disable=protected-access
 Properties.__new__.__defaults__ = (None,) * len(Properties._fields) # type: ignore
```

Comment:
We didn't update the defaults of `Properties` as well?

Reply:
The default was always `None` across the board in Properties as per the definition [here](https://github.com/Azure/azure-sdk-for-python/blob/c15c08437edc64662ff759572b99f7c0ba46ac78/sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/constants.py#L68). The spec also doesn't mention any defaults for the properties.
Did we want to update the defaults to non-None?

Reply:
ah yeah, we don't need line 99